Monitor/Monitor.h

```diff
@@ -744,8 +744,15 @@ POP_WARNING()
             }
             void Activated (const string& callsign, PluginHost::IShell* service) override
             {
+            }
+            void Deactivated (const string& callsign, PluginHost::IShell* service) override
+            {
+            }
+            void Unavailable(const string&, PluginHost::IShell*) override
+            {
+            }
+            void Initialize(const string& callsign, PluginHost::IShell* service) override{
                 /* See comment in the Dispatch method on why no locking is here to protect the _monitor member */
```

Comment:
I think adding this now to Initialize is not correct / could lead to issues. This is now called before plugin initialize has run so for example the query interface to IMemory might not yet succeed as root has not been executed and the IMemory may be coming from the OOP side. I would propose to keep this in the Activated call